Makefile

```diff
@@ -0,0 +1,18 @@
+all: build
+
+dependencies:
+	go get -u github.com/golang/lint/golint
+	go get -u golang.org/x/tools/cmd/cover
+	go get -u golang.org/x/tools/cmd/vet
+
+build:
+	go fmt  ./...
+	golint ./...
+	go vet  ./...
+	go test -v -bench=.
+	go install ./...
+	go test -covermode=atomic -coverprofile=cover.out . \
+	&& go tool cover -html=cover.out -o=cover.html \
+	&& rm cover.out
+
+.PHONY: all dependencies build
```

Comment:
I also don't understand how to fit package coverage into a bash alias. It's fine when it's just 1 layer deep but when you have multiple packages it gets tricker.